docker-entrypoint.sh

```diff
@@ -23,7 +23,7 @@ if [ ! -d /db/db ] ; then
 
     if [ "$OVERPASS_MODE" = "init" ]; then
         lftp -c "get -c \"$OVERPASS_PLANET_URL\" -o /db/planet; exit" \
-        && /app/bin/init_osm3s.sh /db/planet /db/db /app "--meta=$OVERPASS_META" \
+        && /app/bin/init_osm3s.sh /db/planet /db/db /app "--meta=$OVERPASS_META" "--compresion_method=$OVERPASS_COMPRESSION" \
```

Comment:
... and the parameter should be `--compression-method=` with a dash (not an underscore, the usage text is wrong).